packages/core/src/renderers/field.util.ts

```diff
@@ -55,7 +54,7 @@ export const mapStateToFieldProps = (state, ownProps) => {
   const path = composeWithUi(ownProps.uischema, ownProps.path);
   const visible = _.has(ownProps, 'visible') ? ownProps.visible : isVisible(ownProps, state);
```

Comment:
must this not be state.jsonforms?
also the next line?

Reply:
No, since `isVisible` and `isEnabled` do not extract anything directly from the state, only via `getData` which expects the root state.